sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/Mappings.java

```diff
@@ -76,12 +76,14 @@ private static String convertToString(Object value, AttributeType type) {
 
     static <T> String join(List<T> values) {
         StringBuilder sb = new StringBuilder();
-        for (Object val : values) {
-            if (sb.length() > 0) {
+        sb.append("[");
+        for (int i = 0; i < values.size(); i++) {
+            if (i > 0) {
                 sb.append(", ");
             }
-            sb.append(val);
+            sb.append(values.get(i));
         }
+        sb.append("]");
```

Comment:
it's not needed anymore since we compare object instead of string. 
in the json config for array type, we use [value1, value2, value3, ...], string compare needs to match square brackets.
can strip brackets before compare. either way will work.